starter-code/css/main.css

```diff
@@ -115,3 +124,52 @@ width: 1000px;
 .icons img {
   width: 200px;
 }
+
+/*SECTION THREE*/
+
+.yeezy {
+  margin: 0 50px 80px 50px;;
+  background-color: #00B172;
+  display: flex;
+}
+
+.yeezy img {
+  width: 50%;
+  margin: 100px 50px 50px 200px;
+}
+
+.yeezy h2, h3, p {
+  color: white;
+}
+
+.yeezy h2 {
+  font-size: 35px;
+  text-decoration: underline;
```

Comment:
`text-decoration: underline;`is a great idea! In our case a border might be more appropriate though.